tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -437,8 +437,12 @@ class ExecutionTest {
     TEST_METHOD_PROPERTY(L"Priority", L"2") // Remove this line once warp supports this feature in Shader Model 6.3
   END_TEST_METHOD()
 
-  TEST_METHOD(GraphicsRawBufferLdStI16);
-  TEST_METHOD(GraphicsRawBufferLdStHalf);
+  BEGIN_TEST_METHOD(GraphicsRawBufferLdStI16)
+    TEST_METHOD_PROPERTY(L"Priority", L"2") // This test is disabled because of a bug in WARP; TODO: enable once the bug is fixed
```

Comment:
Hmm, this is public code, so does "once the bug is fixed" mean that we've shipped a new version of warp? Just wondering.

Reply:
Yes, I've filed a bug for warp and it should be fixed in the next release.

Reply:
I just meant that "once the bug is fixed" means something different for us vs the public since we get access to new bits ahead of release. Just something to keep in mind when calling a bug fixed.